utils/transformation.py

```diff
@@ -188,4 +216,53 @@ async def image_compression(self, img_path, rate):
             im.save(new_file_name, quality=qlt)
             new_file_size = get_file_units(os.stat(new_file_name).st_size)
         old_file_size = get_file_units(old_file_size)
-        return (old_file_size, new_file_size, new_file_name)
\ No newline at end of file
+        return (old_file_size, new_file_size, new_file_name)
+
+class GifCreate(Command):
```

Comment:
Should probably move these to a separate GIFs cog or something

Reply:
GifAppend.command() exception is now fixed, and gif commands are now moved to a GIF cog.